js/ad_manager_vast.js

```diff
@@ -1921,6 +1941,13 @@ OO.Ads.manager(function(_, $) {
       } else {
         failedAd();
       }
+
+      if (currentAd && currentAd.isAdRequest)
```

Comment:
Is an ad request ad able to make it to the handleAds function? handleAds is only called after we receive the Vast response in the onVastResponse function.